### PR TITLE
Add ZeroTier

### PR DESCRIPTION
**Pull request by jkmartindale**
ZeroTier is a software-defined network + VPN provider ranking #32,016 on [SimilarWeb](https://www.similarweb.com/website/zerotier.com/). Their account system supports TOTP and passkeys.

Unfortunately I wasn't able to find documentation explaining how to set up TOTP/passkeys in ZeroTier. It's just available from their settings portal. I have that URL in the `documentation` field because I wasn't sure if that was necessary or not.

### CODE REVIEW

File: entries/z/zerotier.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "ZeroTier": {
+    "domain": "zerotier.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://accounts.zerotier.com/realms/zerotier/account/#/account-security/signing-in",
```
Comment (Carlgo11) [inline entries/z/zerotier.com.json]: The documentation cannot be behind a login screen. Please remove  the link and post a screenshot of the 2FA setup process or login process.

> [!WARNING]
> Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment like this:
screenshot
```suggestion
```

File: img/z/zerotier.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="86.376" height="86.474" fill="none"><mask id="a" width="87" height="87" x="0" y="0" maskUnits="userSpaceOnUse"><path fill="#fff" d="M86.376 0H0v86.476h86.376z"/></mask><g mask="url(#a)"><path fill="#ffb441" fill-rule="evenodd" d="M0 14.412C0 6.454 6.445 0 14.396 0H71.98c7.95 0 14.396 6.454 14.396 14.412v57.65c0 7.96-6.446 14.412-14.396 14.412H14.396C6.446 86.474 0 80.021 0 72.062Z" clip-rule="evenodd"/></g><path fill="#000" fill-rule="evenodd" d="M74.4 10.214H11.977a2.047 2.047 0 1 0 0 4.095h29.284a2 2 0 0 0-.12.683v6.497c-11.09 1.033-19.774 10.375-19.774 21.75 0 11.373 8.682 20.715 19.775 21.748v9.228a2.049 2.049 0 0 0 4.096 0v-9.228c11.09-1.033 19.775-10.375 19.775-21.749s-8.682-20.716-19.775-21.75v-6.496c0-.24-.043-.47-.119-.682h29.283a2.047 2.047 0 1 0 0-4.096zM25.578 43.238c0-9.043 6.802-16.493 15.564-17.507v35.017C32.38 59.73 25.578 52.283 25.578 43.24Zm35.222 0c0 9.043-6.802 16.493-15.564 17.508V25.728c8.762 1.017 15.564 8.465 15.564 17.51" clip-rule="evenodd"/></svg>
```
Comment (Carlgo11) [inline img/z/zerotier.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 86.376 86.474"><path fill="#ffb441" d="M0 14.412C0 6.454 6.445 0 14.396 0H71.98c7.95 0 14.396 6.454 14.396 14.412v57.65c0 7.96-6.446 14.412-14.396 14.412H14.396C6.446 86.474 0 80.021 0 72.062Z"/><path d="M74.4 10.214H11.977a2.047 2.047 0 1 0 0 4.095h29.284a2 2 0 0 0-.12.683v6.497c-11.09 1.033-19.774 10.375-19.774 21.75 0 11.373 8.682 20.715 19.775 21.748v9.228a2.049 2.049 0 0 0 4.096 0v-9.228c11.09-1.033 19.775-10.375 19.775-21.749s-8.682-20.716-19.775-21.75v-6.496c0-.24-.043-.47-.119-.682h29.283a2.047 2.047 0 1 0 0-4.096zM25.578 43.238c0-9.043 6.802-16.493 15.564-17.507v35.017C32.38 59.73 25.578 52.283 25.578 43.24Zm35.222 0c0 9.043-6.802 16.493-15.564 17.508V25.728c8.762 1.017 15.564 8.465 15.564 17.51"/></svg>
```

File: entries/z/zerotier.com.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "categories": [
+      "remote",
```
Comment (hkamran80) [inline entries/z/zerotier.com.json]: Does ZeroTier provide a remote access platform like GoTo, AnyDesk, ngrok, or TeamViewer?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @up_the_irons!
Just have a few things that needs changing :wink:

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by hkamran80** [comment]
@Carlgo11, we did it again!

**Comment by Carlgo11** [comment]
@hkamran80 you gotta be quicker!
I'm able to do this and code an [SMTP server](https://github.com/Carlgo11/smtp-server) at the same time :stuck_out_tongue_winking_eye:

File: entries/z/zerotier.com.json (hunk 3 above)
Comment (jkmartindale) [reply entries/z/zerotier.com.json]: Similar use case to ngrok where you can allow a local network resource available remotely, though I guess that would be through a software-defined WAN or VPN style setup and not a simple reverse proxy. After comparison with GoTo/AnyDesk/TeamViewer I don't think the remote category fits.

**Comment by jkmartindale** [comment]
image

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**